### PR TITLE
[Edit] HTML Add Media: Elements

### PR DESCRIPTION
**Pull request by cjparsonson**
### Description

Closes issue #3178 
Added a small .png of rendered HTML which shows the HTML tags in the markdown file, rendered in the browser.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by dakshdeepHERE — APPROVED** [review]
Looks good to me

File: content/html/concepts/elements/elements.md [hunk 1]
```diff
@@ -77,3 +77,5 @@ The following example contains four HTML elements: `<html>`, `<body>`, `<h1>`, a
   </body>
 </html>
 ```
+
```
Comment (SSwiniarski) [inline content/html/concepts/elements/elements.md:80]: I'd like a sentence introducing the image.

Comment (cjparsonson) [reply content/html/concepts/elements/elements.md:80]: No problem at all, I'll add that in.

Comment (cjparsonson) [reply content/html/concepts/elements/elements.md:80]: @SSwiniarski does the sentence that I have added resolve your change request?

**Review by SSwiniarski — APPROVED** [review]
@cjparsonson Looks good.

**Pull request merged**